src/util/platform/FFPlatform_unix.c

```diff
@@ -47,6 +47,8 @@ static void getExePath(FFPlatform* platform)
         ssize_t exePathLen = readlink("/proc/self/path/a.out", exePath, sizeof(exePath) - 1);
         if (exePathLen >= 0)
             exePath[exePathLen] = '\0';
+    #elif defined(__HAIKU__)
+        size_t exePathLen = 0; // TODO
```

Comment:
This can be implemented using `get_next_image_info` and searching for the `B_APP_IMAGE` (see https://www.haiku-os.org/legacy-docs/bebook/TheKernelKit_Images.html#get_next_image_info)

Reply:
Yes that's the intent.

Reply:
Actually there's a private get_app_path call that does that.

Reply:
Done.